scripts/ci/jenkins_pipeline_csi

```diff
@@ -25,9 +32,46 @@ pipeline {
                 sh './scripts/run_yamlcheck.sh'
             }
         }
-        stage ('CSI-controller & node: Build and push images') {
+        stage ('Build docker images') {
```

Comment:
could you please unify the three stages into one?

Reply:
why unify them? 
I think that it's more comfortable to see where is the problem from the Jenkins dashboard

Reply:
but we already have enough stages, and these stages are simple and rarely change.
@ArbelNathan what would you prefer?

Reply:
agreed that its to many stages, the debugging process wont change anyway due to fact that the same log will be written minus the the stage name which is too much of a small value for that clutter of stages

Reply:
done